committer/src/config.rs

```diff
@@ -79,13 +82,33 @@ impl Config {
         };
         Ok(algo_config)
     }
+
+    pub fn eth_provider_health_thresholds(&self) -> eth::ProviderHealthThresholds {
+        eth::ProviderHealthThresholds {
+            transient_error_threshold: self.eth.failover.transient_error_threshold,
+            mempool_drop_threshold: self.eth.failover.mempool_drop_threshold,
+            mempool_drop_window: self.eth.failover.mempool_drop_window,
+        }
+    }
+
+    pub fn eth_tx_config(&self) -> eth::TxConfig {
+        eth::TxConfig {
+            tx_max_fee: u128::from(self.app.tx_fees.max),
+            send_tx_request_timeout: self.app.send_tx_request_timeout,
+            acceptable_priority_fee_percentage: eth::AcceptablePriorityFeePercentages::new(
+                self.app.tx_fees.min_reward_perc,
+                self.app.tx_fees.max_reward_perc,
+            )
+            .expect("already validated via `validate` in main"),
```

Comment:
Opinion: Ideally we'd leverage the type system to guarantee this property, as validation logic is prone to change over time. While I think this is fine for now, in similar situations I'd consider defining a `ValidatedConfig` struct that holds the converted types directly and have `validate` convert to this struct.